### ISSUE TITLE
Understanding d-sne and application

### ISSUE THREAD

**Issue by lchunleo** [joshuacwnewton/d-SNE-PyTorch#4]
Hi

  Thanks for sharing your work. I have two questions:

1) Based on your code, it seem one network is used to be trained (LeNet) whereas in the paper, the source and target has their separate networks , where the learning takes place to max-min the loss based on their generated features. I may have misread your code or the paper. Could you advise me?

2) am i correct to read that after the performing the training, the saved checkpoint weights of the e.g. LeNet has been "adapted" to able to use to classify object (without the domain adaption codes) ? Thank you
